packages/native/src/components/Layout/List/List/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components/native";
+import Text from "../../../Text";
+import { Box, Flex } from "../../index";
+import baseStyled, { BaseStyledProps } from "../../../styled";
+import { FlatList } from "react-native";
+
+export type BaseListItemProps = {
+  title?: string;
+  description?: string;
+  bullet?: React.ReactNode;
+};
+
+export type BaseListProps = {
+  items: BaseListItemProps[];
+  itemContainerProps?: BaseStyledProps;
+};
+
+const ListContainer = styled.View`
+  display: flex;
```

Comment:
Why about using a `Flex` instead of a `View` here?

Reply:
Of course, done